internals/containerd.sh

```diff
@@ -0,0 +1,46 @@
+CONTAINERD_VERSION="v1.2.7"
+
+github_force() {
```

Comment:
Same as runc.sh

Reply:
I based my code on the cores.sh builder and it defines it's own `github_force` function.